docs/lightning/rtl.md

```diff
@@ -1,5 +1,83 @@
 # Ride the lightning (RTL)
 
-GUI to manage lightning channels
+A GUI to manage lightning channels
 
-work in progress
+## Using Ride the Lightning (RTL) Wallet
+
+### Logging into RTL
+
+Ride the Lightning is a Lightning wallet and node management tool accessible via a web interface, and is built into myNode.
+RTL is a creation of Suheb/saubyk, and complete information can be found at [Github](https://github.com/ride-the-lightning/RTL).
+The developer can be contacted via Twitter at: @suhub__ or @RTL_App.
+
+You can use RTL from any browser that is able to access your myNode installation.
+
+If you are accessing RTL from inside the network on which your myNode installation resides, you can open a browser window and enter <your.mynode.network.IP:3010> into the address bar.
```

Comment:
maybe we don't need this line since we have mentioned the port numbers in the spec below. Just explain the GUI steps to enable and access it.

Reply:
I agree.